server/src/main/java/org/opensearch/cluster/node/DiscoveryNode.java

```diff
@@ -412,6 +413,21 @@ public Map<String, String> getAttributes() {
         return this.attributes;
     }
 
+    /**
+     * Remote Store segment and translog repository names
+     *
+     * @return {@link Map} of the segment and translog repository with their names
+     */
+    public Map<String, String> getRemoteStoreRepoNames() {
+        assert isRemoteStoreNode();
+        return Map.of(
+            RemoteStoreNodeAttribute.REMOTE_STORE_SEGMENT_REPOSITORY_NAME_ATTRIBUTE_KEY,
+            attributes.get(RemoteStoreNodeAttribute.REMOTE_STORE_SEGMENT_REPOSITORY_NAME_ATTRIBUTE_KEY),
+            RemoteStoreNodeAttribute.REMOTE_STORE_TRANSLOG_REPOSITORY_NAME_ATTRIBUTE_KEY,
+            attributes.get(RemoteStoreNodeAttribute.REMOTE_STORE_TRANSLOG_REPOSITORY_NAME_ATTRIBUTE_KEY)
+        );
+    }
```

Comment:
Move this to `RemoteStoreNodeAttribute` .

Reply:
@sachinpkale/ @shourya035   : Need to take care of  https://github.com/opensearch-project/OpenSearch/pull/12993/files as well . This doesn't look future proof , as we keep adding more repositories . Can we think of something better here ?